astroquery/cadc/tests/test_cadctap_remote.py

```diff
@@ -206,6 +206,7 @@ def test_authsession(self):
 
     @pytest.mark.skipif(one_test, reason='One test mode')
     @pytest.mark.skipif(not pyvo_OK, reason='not pyvo_OK')
+    @pytest.mark.skip('https://github.com/astropy/astroquery/issues/2325')
```

Comment:
I feel leaving these as failing tests are better, it ensures that they are not forgotten. Or maybe change them to xfail, so then we would notice easily if something got fixed upstream.